content/en/security/application_security/software_composition_analysis/_index.md

```diff
@@ -52,40 +52,48 @@ Check the ASM Compatibility for each ASM product to see if your service is suppo
 
 ## Explore and manage SCA vulnerabilities
 
 <div class="alert alert-info">Datadog Software Composition Analysis can find vulnerable libraries across the software development lifecycle (SDLC). Application Security summarizes results found in the default branches of your repositories and in your running services. To view vulnerabilities found in different branches and commits, see <a href="/code_analysis/software_composition_analysis" target="_blank">Code Analysis</a> for more details.</div>
 
-The [Vulnerability Explorer][3] shows a complete list of the open source libraries detected by Datadog SCA and reports security vulnerabilities associated with them. 
+The [Vulnerability Explorer][3] shows a complete list of the open source libraries detected by Datadog SCA and reports security vulnerabilities associated with them.
 
-Datadog SCA leverages two techniques to analyze your services: 
+Datadog SCA leverages two techniques to analyze your services:
 
 - Static code analysis in your repositories (static point of view)
 - Runtime analysis in your deployed services (runtime point of view)
 
 Combining both techniques monitors open source libraries end-to-end, from the code repository commit (static point of view), to the applications running in production (runtime point of view).
 
-To switch to the code repository commit point of view, select **Static**. The static view shows vulnerabilities from the _source code_ in your repositories. 
+To switch to the code repository commit point of view, select **Static**. The static view shows vulnerabilities from the _source code_ in your repositories.
 
 To switch to the _real-time_ point of view for the applications already running, select **Runtime**. The runtime view is the live view of the services monitored by Datadog.
 
 {{< img src="/security/application_security/software_composition_analysis/asm_sca_vulnerabilities.png" alt="Software Composition Analysis (SCA) explorer page showing vulnerabilities sorted by static or runtime." style="width:100%;" >}}
 
-Select a specific vulnerability to see its details, including the affected services, severity breakdown score, and recommended remediation steps. 
+Select a specific vulnerability to see its details, including the affected services, severity breakdown score, and recommended remediation steps.
 
 On the Details Explorer for a vulnerability, you can view impacted infrastructure. This view gives you better insights to your overall attack exposure.
 
-Within ASM, the vulnerability severity base score is modified using existing attacks and the business sensitivity of the environment where the vulnerability is detected. For example, if no production environment is detected, the severity is reduced.
+## Datadog Severity Score
+
+Each vulnerability has a defined base severity score. To help prioritize remediation, Datadog provides an enhanced severity score (known as "Datadog Severity Score") using existing attacks
```

Comment:
"Each vulnerability has a defined base severity score from [CVSS](https://www.first.org/cvss/)."

Reply:
the score is not from CVSS. CVSS is the method

Reply:
I agree with Kassen, we should clearly state that the base score is following CVSS.

Also, I often heard customers didn't understand the Datadog score is following CVSS as well, we should state it clearly.

For example:
`Datadog adjust the base CVSS vector by modifying the temporal and environmental score.`

We could demystify further by explaining how the base CVSS vector is adjusted, modifier by modifier.

For example:
Considering [this CVSS](https://www.first.org/cvss/calculator/3.0#CVSS:3.0/AV:N/AC:H/PR:N/UI:R/S:C/C:L/I:L/A:L/E:H/RL:U/RC:C/CR:H/IR:H/AR:H/MAV:N/MAC:H/MPR:N/MUI:R) having
 * a vector `AV:N/AC:H/PR:N/UI:R/S:C/C:L/I:L/A:L/E:H/RL:U/RC:C/CR:H/IR:H/AR:H/MAV:N/MAC:H/MPR:N/MUI:R`
 * a score of 7.2
 * a `High` severity

If the vulnerability is not detected on a **production environment**, then the environmental score is updated as follow:
 - The Confidentiality Requirement (CR) is set to `Low`
 - The Integrity Requirement (IR) is set to `Low`
 - The Availability Requirement (AR) is set to `Low`

The Datadog score results in:
 * [this CVSS](https://www.first.org/cvss/calculator/3.0#CVSS:3.0/AV:N/AC:H/PR:N/UI:R/S:C/C:L/I:L/A:L/E:H/RL:U/RC:C/CR:L/IR:L/AR:L/MAV:N/MAC:H/MPR:N/MUI:R)
 * having this vector `AV:N/AC:H/PR:N/UI:R/S:C/C:L/I:L/A:L/E:H/RL:U/RC:C/CR:L/IR:L/AR:L/MAV:N/MAC:H/MPR:N/MUI:R`
 * a score of 4.0
 * a `Medium` severity


@kassenq @juli1 what do you think?

If we want to follow this path, @juli1 you can see how the CVSS is adjusted in [this class](https://github.com/DataDog/logs-backend/blob/a0c521c50875a7863c4a4648c98d1fdf982ba437/domains/cloud-security-platform/application-security/libs/vulnerability-common/src/main/java/com/dd/appsec/vulnerability/cvss/CvssModifier.java#L122)

Reply:
CVSS defines how to change the vuln attributes in score (https://www.first.org/cvss/v3.1/specification-document#1-2-Scoring) and how to map this score to severity (https://www.first.org/cvss/v3.1/specification-document#Qualitative-Severity-Rating-Scale)

Reply:
I would say that the Datadog score also leverages CVSS